src/errors/runtime/type-assertions.js

```diff
@@ -2,8 +2,17 @@ import { isFinite, isRegExp, isNil as isNullOrUndefined } from 'lodash';
 import { APIError } from './';
 import MESSAGE from './message';
 
+function isNumber (value) {
```

Comment:
Because we need exactly `isFinite` check https://lodash.com/docs/4.17.2#isNumber.
But we can simplify it